backend/src/main/java/com/example/backend/model/writing/Suggestion.java

```diff
@@ -0,0 +1,67 @@
+package com.example.backend.model.writing;
+
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "suggestion")
+public class Suggestion {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "id_generator")
+    @SequenceGenerator(name="id_generator", sequenceName = "suggestion_id_seq", allocationSize = 1)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "language_id")
+    private Integer languageId;
+
```

Comment:
How will they push language id? Will the page send another request to fetch all languages from language endpoint?

Reply:
They can use /lang endpoint to get all ids.

Reply:
Okay nice.